egs/wsj/s5/steps/nnet/train_mmi.sh

```diff
@@ -68,6 +68,10 @@ if ! $skip_cuda_check; then cuda-compiled || { echo "Error, CUDA not compiled-in
 
 mkdir -p $dir/log
 
+utils/lang/check_phones_compatible.sh $lang/phones.txt $srcdir/phones.txt
+utils/lang/check_phones_compatible.sh $lang/phones.txt $alidir/phones.txt
```

Comment:
I think the lines is different. Both srcdir and alidir are the input directories. So I check both of them. Do I really need to delete a line? I will fix the "|| exit 1" bug , if a script has set -e in it.